GOVERNACE.md

```diff
@@ -1,45 +1,87 @@
+> **Note**
+>
+> This is a **Work in Progress**. Stay tuned for more follow-up updates!
+
 # Governace
 
+The TiKV community is a meritocratic, consensus-based community project. Anyone with an interest in the project can join the community, contribute to the project design and participate in the decision making process. This document describes how that participation takes place and how to set about earning merit within the project community.
+
 ## Code of Conduct
 
 The TiKV community follows the TiKV Code of Conduct. Here are some excerpts:
 
-> In the interest of fostering an open and welcoming evironment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
 
-## Community Membership
+## Community Roles and Responsibilities
 
 Refer to [Community Membership](community-membership.md).
 
 ## Community Structure
 
 ![community organization](/media/governace/community_organization.png)
 
-The community structure comprises two parts. The first part is the community organization, and the second part is the role division within the community group and the promotion path of each community role.
+The project is comprised of the following types of subgroups:
+
+- Project Management Committee, PMC
+- Organizer Committee
```

Comment:
Maintainers?

Reply:
Maintainer is a role under Developer Group, and is detailed in Membership.  Here we talk about the subgroup levels, which include the listed groups. Does it make sense to you?